reactnative/ReactNativeTemplateApp/src/com/salesforce/samples/reactnativetemplateapp/ReactNativeTemplateApp.java

```diff
@@ -28,14 +28,41 @@
 
 import android.app.Application;
 
+import com.facebook.react.ReactApplication;
+import com.facebook.react.ReactNativeHost;
+import com.facebook.react.ReactPackage;
+import com.facebook.react.shell.MainReactPackage;
 import com.salesforce.androidsdk.app.SalesforceSDKManager;
 import com.salesforce.androidsdk.reactnative.app.SalesforceReactSDKManager;
 import com.salesforce.androidsdk.security.Encryptor;
 
+import java.util.Arrays;
+import java.util.List;
+
 /**
  * Application class for our application.
  */
-public class ReactNativeTemplateApp extends Application {
+public class ReactNativeTemplateApp extends Application implements ReactApplication {
+
+	private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
+		@Override
+		protected boolean getUseDeveloperSupport() {
+			return BuildConfig.DEBUG;
```

Comment:
We don't need it in the activity.
ReactActivity will get the value from the application. See https://github.com/facebook/react-native/blob/master/ReactAndroid/src/main/java/com/facebook/react/ReactActivity.java#L89
I'll fix it.